rice-namenode/source/main.cc

```diff
@@ -60,6 +62,13 @@ static inline int parse_cmdline_options(
           return -1;
         }
         break;
+      case 'n':*node_policy = optarg[0];
+        if (*node_policy != MIN_XMITS || *node_policy != MAX_FREE_SPACE) {
+          LOG(ERROR) << "Node policy must be -x (minimum transmits) or "
```

Comment:
seems like the usage is `-n x` or `-n f` or nothing. if that's true you might want to change this error message.